console/services/market_app/component.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf8 -*-
+import logging
+from datetime import datetime
+
+# service
+from console.services.app_config import env_var_service
+# repository
+from console.repositories.app_config import port_repo
+# model
+from www.models.main import TenantServicesPort
+from www.models.main import TenantServiceEnvVar
+from www.models.main import TenantServiceVolume
+from console.models.main import ComponentGraph
+from www.models.main import TenantServiceConfigurationFile
+# exception
+from console.exception.main import EnvAlreadyExist, InvalidEnvName
+# enum
+from console.enum.app import GovernanceModeEnum
+# util
+from www.utils.crypt import make_uuid
+
+logger = logging.getLogger("default")
+
+
+class Component(object):
+    def __init__(self,
```

Comment:
Component dependencies, volume dependencies, plugin dependencies,  configuration group dependencies are not handled here?